libs/shared/src/lib/theme/theme.ts

```diff
@@ -4,14 +4,14 @@ export const theme = ( mode?: PaletteMode ): Theme => createTheme({
   palette: {
     mode: mode,
     primary: {
-      main: '#43c48e',
-      light: '#7bd6b0',
+      main: '#5fa800',
+      light: 'rgba(95,168,0,0.6)',
       dark: '#287655',
     },
     success: {
       main: '#00C256',
       light: 'rgba(0, 178, 79, 0.12)',
```

Comment:
did you use material theming generator to generate the color for success based on main variants

Reply:
this i will redo after